gcp-logging/build.gradle

```diff
@@ -6,8 +6,8 @@ dependencies {
     compileOnly(projects.micronautGcpTracing)
     api(projects.micronautGcpCommon)
     implementation(libs.logback.json.classic)
+    implementation(mnSerde.micronaut.serde.support)
```

Comment:
should this be `micronaut.micronaut.json` ?

Reply:
I am making this change, but note that the gcp-logging module (as well as gcp-tracing) are still disabled in settings.gradle.